.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
+
+Please make sure you have read our coding guidelines: https://github.com/mlr-org/mlr/wiki/mlr-Coding-Guidelines
+
+Also it's helpful to find a buddy from the core team who helps you getting your PR merged.
```

Comment:
Maybe link to [this](https://github.com/mlr-org/mlr/wiki/mlr-developer-team-and-contributors-page) list? However, these are all contributors and not just the 'core' guys (I do not know myself who the core guys are)